controllers/selfnoderemediation_controller.go

```diff
@@ -169,7 +177,7 @@ func (r *SelfNodeRemediationReconciler) isFencingCompleted(snr *v1alpha1.SelfNod
 	return snr.Status.Phase != nil && *snr.Status.Phase == fencingCompletedPhase && snr.DeletionTimestamp != nil
 }
 
-func (r *SelfNodeRemediationReconciler) remediateWithResourceDeletion(snr *v1alpha1.SelfNodeRemediation) (ctrl.Result, error) {
+func (r *SelfNodeRemediationReconciler) remediateWithResourceRemoval(snr *v1alpha1.SelfNodeRemediation, removeResource func(*v1.Node, *v1alpha1.SelfNodeRemediation) (time.Duration, error)) (ctrl.Result, error) {
```

Comment:
Nit: I think that the name `removeResource` might be a bit confusing.
Maybe something like `removeNodeResources` or even `removeResources` ?

Reply:
Changed to removeNodeResources. Thanks.